assets/shaders/octree/coneTracing.glsl

```diff
@@ -150,7 +161,7 @@ void main() {
     }
 
     if (shouldShowDirect) {
-        finalImage += vec4(visibility * directLight, 1.0);
+        finalImage += visibility * vec4(directLight, 1.0);
```

Comment:
We don't want to multiplicate the alpha by the visibility. If it is in shadow it should be black, not transparent